climada/engine/test/test_impact_calc.py

```diff
@@ -135,6 +135,27 @@ def test_error_handling_mismatch_haz_type(self):
         except Exception as e:
             self.assertEqual(str(e), "Impact calculation not possible. No impact "
                              "functions found for hazard type TC in impf_set.")
+    def test_error_handling_mismatch_impf_ids(self):
```

Comment:
If being pedantic, I would improve slightly the test by having two impact functions in the set, one that is in the exposures, and one that isn't.

Reply:
Ok so now the test considers the case of having two impact functions in the exposures (impf ids 1 and 2), and two impact functions in the impfset (impf ids 1 and 3), so that an error is raised as there is no impact function with id 2 in the impfset. Let me know in case you meant something else.